client/me/purchases/concierge-banner/index.jsx

```diff
@@ -42,86 +42,91 @@ class ConciergeBanner extends Component {
+					`Schedule your {{supportLink}}Quick Start support session{{/supportLink}} and get one-on-one guidance from our expert Happiness Engineers to kickstart your site.`,
 					{
-						comment:
-							'Quick Start Session is a one-on-one video session between the user and our support staff.',
-						args: { durationInMinutes: 30 },
+						components: {
+							supportLink: (
+								<a
+									target={ '_blank' }
+									rel={ 'noreferrer' }
+									href="https://wordpress.com/discover-wordpress/2019/03/21/getting-the-most-out-of-our-business-concierge-service/"
+								/>
+							),
+						},
 					}
 				);
-				buttonText = translate( 'Schedule now' );
-				buttonHref = '/me/quickstart';
-				illustrationUrl = conciergeImage;
-				break;
-
-			case CONCIERGE_HAS_AVAILABLE_PURCHASED_SESSION:
-				headerText = translate( 'Our experts are waiting to help you' );
-				mainText = translate( 'Schedule your 1-1 Quick Start Session with a Happiness Engineer!', {
-					comment:
-						'Quick Start Session is a one-on-one video session between the user and our support staff.',
-				} );
-				buttonText = translate( 'Schedule now' );
-				buttonHref = '/me/quickstart';
-				illustrationUrl = conciergeImage;
+				buttonText = translate( 'Schedule a date' );
+				event = 'schedule-concierge-session';
 				break;
 		}
 
-		return { headerText, mainText, buttonText, buttonHref, illustrationUrl };
+		return { title, description, buttonText, event };
 	}
 
 	render() {
-		const { showPlaceholder } = this.props;
+		const { bannerType, showPlaceholder, nextAppointmentSiteId, siteId, availableSessions } =
+			this.props;
+
+		// if no appointment and not on a site then use the first site with a session available
+		// e.g. when viewing at /me/purchases
+		const quickStartSite = nextAppointmentSiteId || siteId || availableSessions[ 0 ];
 
 		if ( showPlaceholder ) {
 			return this.placeholder();
 		}
 
-		const { headerText, mainText, buttonText, buttonHref, illustrationUrl } =
-			this.getBannerContent();
+		const { buttonText, description, title, event } = this.getBannerContent();
+
+		const referrer = location.pathname.startsWith( '/me/purchases' )
+			? '/me/purchases'
+			: '/purchases/subscriptions/:site';
 
 		return (
 			<>
 				<TrackComponentView eventName="calypso_purchases_concierge_banner_view" />
-				<ActionCard
-					headerText={ headerText }
-					mainText={ mainText }
-					buttonText={ buttonText }
-					buttonIcon={ null }
-					buttonPrimary={ true }
-					buttonHref={ buttonHref }
-					buttonTarget={ null }
-					buttonOnClick={ () => {
-						this.props.recordTracksEvent( 'calypso_purchases_concierge_banner_click', {
-							referer: '/me/purchases',
-						} );
+				<Banner
+					showIcon={ false }
+					primaryButton={ false }
+					callToAction={ buttonText }
+					description={ description }
+					dismissPreferenceName={ `quick-start-banner-${ bannerType }` }
+					href={ `/me/quickstart/${ quickStartSite }` }
```

Comment:
Might it be cleaner to explicitly adjust the path based on whether `quickStartSite` is defined? e.g. 
```
quickStartUrl = '/me/quickstart' . ( quickStartSite ? `/${ quickStartSite }` : '' );
```
My concern here is that I am not sure the URL with a trailing slash will be redirected/handled correctly.

Reply:
Follow-up: I think this should generally be the site slug instead of the siteId. So we should use the `getSiteSlug()` selector based on the value of `siteId` to build the URL. The site ID works, but we almost always use the site slug in our URLs.

Reply:
We should always have a `quickStartSite` when this component gets rendered now, because of the `shouldShowBanner` check in `<PurchaseListConciergeBanner/>`. There is a good case to merge these two components into one, as it would make the logic clearer. Maybe that could be done in the next iteration. 

I have made it use the slug for now.

Reply:
The code is now using `getSiteSlug`. There is no need for conditional logic here as we will always have a site if the Banner is shown.